netsuite/10.0/modules/ROOT/pages/netsuite-reference.adoc

```diff
@@ -990,7 +990,7 @@ Used to change the email address for the account.
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| New Email a| String |  New email value. |  `#[payload]` |
+| New Email a| String |  New email value. |  |
```

Comment:
```suggestion
| New Email a| String |  New email value |  |
```
Should we fill empty cells with "not applicable" (or "n.a."), so that they know that the cell was not accidentally left empty?

Reply:
That's a conversation for another day. This is generated, so I'm not sure what's possible.